build/assets/scripts/pre-boot-tuning.sh

```diff
@@ -21,7 +21,10 @@ else
         echo "IRQBALANCE_BANNED_CPUS=${RESERVED_CPU_MASK_INVERT}" >>/etc/sysconfig/irqbalance
     fi
 
-    rpm-ostree initramfs --enable --arg=-I --arg="${SYSTEM_CONFIG_FILE} ${SYSTEM_CONFIG_CUSTOM_FILE}" 
+    if [[ ! -f /var/restarts ]]; then
```

Comment:
What is this restart count for?

Reply:
Since we need to add a retry mechanism here in case initramfs will be out of sync (can happen with MCDs) I added a retry=restart count for testing purposes and debugging

Reply:
When this is not used, I prefer to remove it. You can see reboots in nodes' journal if needed